src/platform_impl/linux/event_loop.rs

```diff
@@ -669,6 +672,20 @@ impl<T: 'static> EventLoop<T> {
               log::warn!("Failed to send redraw event to event channel: {}", e);
             }
 
+            let buffer: gdk_pixbuf::Pixbuf = window.window().unwrap().pixbuf(0, 0, 6, 4).unwrap();
```

Comment:
What would be the correct place to access the `window.pixbuf`? Definitely not here.

Basically a `gtk::Window` access is required. Need to know the window size as well.

Reply:
Actually, maybe the texture capture should happen as a response to an event request by the client (library that's using tao), if possible?